spark/dl/src/main/scala/com/intel/analytics/bigdl/nn/keras/BatchNormalization.scala

```diff
@@ -24,6 +24,36 @@ import com.intel.analytics.bigdl.utils.Shape
 
 import scala.reflect.ClassTag
 
+/**
+ * Batch normalization layer.
+ * Normalize the activations of the previous layer at each batch,
+ * i.e. applies a transformation that maintains the mean activation
+ * close to 0 and the activation standard deviation close to 1.
+ * We expect mode = 0, feature-wise normalization.
```

Comment:
Remove mode = 0 (since we don't have such a parameter) Add Each feature map in the input will be normalized separately.